modules/localstack/src/main/java/org/testcontainers/containers/localstack/LocalStackContainer.java

```diff
@@ -120,9 +145,13 @@ private static boolean shouldRunInLegacyMode(String version) {
     protected void configure() {
         super.configure();
 
-        Preconditions.check("services list must not be empty", !services.isEmpty());
+        if (servicesListRequired) {
+            Preconditions.check("services list must not be empty", !services.isEmpty());
+        }
 
-        withEnv("SERVICES", services.stream().map(EnabledService::getName).collect(Collectors.joining(",")));
+        if (!services.isEmpty()) {
+            withEnv("SERVICES", services.stream().map(EnabledService::getName).collect(Collectors.joining(",")));
+        }
```

Comment:
> Scripts that depend on the output of /health being running will require setting setting EAGER_SERVICE_LOADING=1, which will start all services eagerly. This works together with the SERVICES variable to create the behavior from localstack <= 0.13.0.

Following the note from the release notes and do not break things


```suggestion
            if (this.servicesEnvVarRequired) {
                withEnv("EAGER_SERVICE_LOADING", "1");
            }
        }
```